script.js

```diff
@@ -256,6 +251,17 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 
+  
+});
+
+form.addEventListener('input', (event) => {
+  event.preventDefault();
+
+  const fullName = fullNameInput.value.trim();
+  const email = emailInput.value.trim();
+  const firstName = firstNameInput.value.trim();
+  const lastName = lastNameInput.value.trim();
+
```

Comment:
- Great job so far on implementing the changes proposed by the previous reviewer, however, I just want to explain something in detail as the previous reviewer had given a summary of what you need to do.

- [ ] Right now, all your input data get saved in localStorage when I reload without submitting except for the `textarea` input which disappears with and without submitting the form. What you want to achieve is to be able to have all the inputs value saved to the localStroge when the user reloads the page without submitting the form and also when the user submits the form. vise versa. As of now, only the textarea is missing this part of the requirement. Please, try and fix this issue then submit it for another review.
- [ ] [OPTIONAL BUT REQUIRED] *NOTE: I also observed that this is a pair-programming project, but you've worked on it solo. Please, kindly ensure we put this in mind for the subsequent projects*

Reply:
@zieeco the previous reviewer didn't mention anything about the text area and you could have commented on that as an optional. Imagine I am repeating the week because of this. Thank you bro, 👏